requirements.txt

```diff
@@ -3,11 +3,12 @@ coveralls==0.5
 Django==1.7.4
 docopt==0.6.2
 flake8==3.4.1
-psycopg2==2.5.4
+psycopg2==2.7.3
 PyYAML==3.11
 requests==2.7.0
-selenium==2.53.1
+selenium==3.8.0
```

Comment:
:wave: Did Travis pass for your fork after this change?

Reply:
No it states that geckodriver is needed with this version of selenium.

Reply:
thought so, then we revert this change.

Reply:
Should I change selenium version back to 2.53.1?